proposals/0000-reduce-fragmentation.md

```diff
@@ -0,0 +1,837 @@
+
+* [ ] `calc()`(https://developer.mozilla.org/en-US/docs/Web/CSS/calc)
+* [ ] `clamp()`(https://developer.mozilla.org/en-US/docs/Web/CSS/clamp)
+* [ ] `max()`(https://developer.mozilla.org/en-US/docs/Web/CSS/max)
+* [ ] `min()`(https://developer.mozilla.org/en-US/docs/Web/CSS/min)
+* [ ] `minmax()`(https://developer.mozilla.org/en-US/docs/Web/CSS/minmax)
+* [ ] `url()`(https://developer.mozilla.org/en-US/docs/Web/CSS/url)
+* [ ] `var()`(https://developer.mozilla.org/en-US/docs/Web/CSS/var)
+
+## CSS Lengths
+
+Support major units where supported by CSS.
+
+* [ ] `em` units.
+* [ ] `rem` units.
+* [ ] `px` units.
+* [ ] `v*` units.
+* [ ] `%` units.
+
+## CSS Media Queries
+
+Support CSS Media Queries. Although Media Queries are not a preferred long-term solution, [Container Queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries) are not yet widely supported by browsers. This could be shimmed in user-space on top of the existing `StyleSheet` API.
+
+## CSS Properties and Values
+
+Miscellaneous CSS properties and values that should be supported on native.
+
+* [ ] `backgroundImage`. Add support for setting background images via `url()`. Stretch: support CSS gradients.
+* [ ] `backgroundPosition`.
+* [ ] `boxSizing`.
+* [ ] `boxShadow`. Add native support for CSS box shadows to replace buggy, iOS-specific `shadow*` styles.
+* [ ] `caretColor`.
+* [ ] `clipPath`.
+* [ ] `display` values of `block`, `inline`, `inline-block`, `inline-flex`.
+* [ ] `float`.
+* [ ] [`justifySelf`](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-self). Sets the way a box is justified inside its alignment container along the appropriate axis.
+* [ ] (`lineClamp` is equivalent to `numberOfLines` prop on `<Text>` components).
+* [ ] [`objectPosition`](https://developer.mozilla.org/en-US/docs/Web/CSS/object-position).
+* [ ] `overflowBlock`.
+* [ ] `overflowInline`.
+* [ ] `overflowX`.
+* [ ] `overflowY`.
+* [ ] [`placeContent`](https://developer.mozilla.org/en-US/docs/Web/CSS/place-content). Shorthand for setting `alignContent` and `justifyContent`. Value is an enum of `baseline`, `first baseline`, `last baseline`, `center`, `end`, `flex-start`, `flex-end`, `left`, `right`, `safe`, `space-around`, `space-between`, `space-evenly`, `start`, `stretch`, `unsafe`. Note that `space-evenly` should also be supported for `alignContent` and `justifyContent`.
+* [ ] [`placeItems`](https://developer.mozilla.org/en-US/docs/Web/CSS/place-items). Shorthand for setting `alignItems` and `justifyItems`. Value is an enum of `auto`, `baseline`, `first baseline`, `last baseline`, `center`, `end`, `flex-start`, `flex-end`, `left`, `right`, `start`, `stretch`.
+* [ ] [`placeSelf`](https://developer.mozilla.org/en-US/docs/Web/CSS/place-self). Shorthand for setting `alignSelf` and `justifySelf`. Value is an enum of `auto`, `baseline`, `first baseline`, `last baseline`, `center`, `end`, `flex-start`, `flex-end`, `left`, `normal`, `right`, `start`, `stretch`.
```

Comment:
These might be relatively easy to implement (other than the full list of values) as they expand into properties with existing support.